converter/internal/staticconvert/staticconvert.go

```diff
@@ -31,7 +31,7 @@ func Convert(in []byte, extraArgs []string) ([]byte, diag.Diagnostics) {
 	var diags diag.Diagnostics
 
 	fs := flag.NewFlagSet("convert", flag.ContinueOnError)
-	args := []string{"-config.file", "convert", "-config.expand-env"}
+	args := []string{"-config.file", "convert"}
```

Comment:
Remove this hard coded since we can now pass this in and have more control over the converter. This was the lesser of 2 evils before to automatically expand environment vars.